benchopt/utils/shell_cmd.py

```diff
@@ -3,14 +3,19 @@
 import subprocess
 
 from ..config import DEBUG
-from ..config import get_setting
+
+
+def get_setting(setting_name):
+    settings = {
+        'shell': 'bash' if os.name != 'nt' else 'cmd'  # Default to cmd on Windows
+    }
+    return settings.get(setting_name, 'bash')
 
 
 SHELL = get_setting('shell')
 
 
-def _run_shell(script, raise_on_error=None, capture_stdout=True,
```

Comment:
@wassimmazouz avoid cosmetic changes like this, they are not related to the changes you introduce but make the diff longer hence harder to review. Your autoformatter is probably the culprit here